ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/router-bgp.md

```diff
@@ -939,6 +957,24 @@
         networks:
           - prefix: <str>
             route_map: <str>
+      address_family_path_selection:
+        bgp:
+          missing_policy:
+            direction_in_action: <str>
+            direction_out_action: <str>
+          additional_paths:
+            receive: <bool>
+        neighbors:
+          - ip_address: <str>
+            activate: <bool>
+            route_map_in: <str>
+            route_map_out: <str>
+        peer_groups:
+          - name: <str>
+            activate: <bool>
+        networks:
+          - prefix: <str>
+            route_map: <str>
```

Comment:
Please make sure EOS supports all these options under this address-family.

Reply:
Removing because networks is not supported under this address-family will resubmit.